DETAILED_INSTRUCTIONS.md

```diff
@@ -52,7 +54,7 @@ git clone https://github.com/Juniper/yang.git
 
 *Note - this may take some time*
 
-Next, to match one of Rahul's examples, let's copy a YANG models into working directory from the Juniper hoard of models.
+Next, to match one of Rahul's examples, let's copy a YANG model into working directory from the Juniper hoard of models.
```

Comment:
remove personal names and give specifics of which example to refer. Add a link to https://github.com/Juniper/yang for Juniper yang models.

Reply:
I've removed the personal names, included an additional link for the YANG repository and simplified the language around samples/example etc.